lib/Table/TableStyle.ts

```diff
@@ -399,3 +404,5 @@ export default class TableStyle {
     return this.tableModel.tableColumns.find(column => column.name === name);
   }
 }
+
+getColorForId;
```

Comment:
Typo, I think. Either that or deeply disturbing side-effect 😉

Reply:
Haha fixed :)